plugin/pki/vcert.go

```diff
@@ -72,18 +111,33 @@ func (c venafiConnectionConfig) getConnection() (endpoint.Connector, error) {
 			}
 			cfg.ConnectionTrust = string(trustBundle)
 		}
+	} else if c.URL != "" && c.AccessToken != "" {
```

Comment:
@angelmoo does this mean that if I specify `url`, `access_token`, `tpp_username`, and `tpp_password` when I create/update the venafi-policy it is going to use the username/password?  If so, it should be the opposite or better yet it would fail with an invalid configuration error because you shouldn't be able to ambiguous credentials.

Reply:
Fixed on las commit, now access token have precedence against user/password, and a validation for mixed credentials  were added just to make monitor more robust.